christfellowshipchurch/web-app
# [Group] Overhaul Group page layout

# Related Issue
- [**🏕 Make group page layout responsive**](https://3.basecamp.com/3926363/buckets/17566548/todos/3185252598)

# Changes or Updates
- Duplicates, then overhauls the group layout _on the sandbox version_ of the site
- Adds members list with a "See all" modal

# Tradeoffs/Caveats
This is still somewhat of a WIP layout.
There are some fairly obvious refinements that could be made to responsive re-stacking of elements, vertical alignments, and the polish/finessing on the new Members List.
This PR/scope is intended to improve what's there now, and add more information missing from the web currently for groups.

# Test Instructions
- Verify individual groups pages load and generally function properly both _with_ and _without_ sandbox mode enabled (`REACT_APP_ENABLE_SANDBOX=true` / `REACT_APP_ENABLE_SANDBOX=`)
- Verify responsive sizes are generally acceptable
- Verify Add to Calendar event description/config is generally acceptable (note: existing issue with Apple calendar on Chrome for some users)

# Screenshots

|Description|Screenshot|
|---|---|
|Sandbox mode disabled|<img src="https://user-images.githubusercontent.com/1812955/98626563-082b3480-22e0-11eb-8e10-e8c2aaa18340.jpg" width="400" />|
|New Layout - **Small**|<img src="https://user-images.githubusercontent.com/1812955/98626631-22651280-22e0-11eb-9dd2-aad41fe2635e.jpg" width="400" />|
|New Layout - **Medium**|<img src="https://user-images.githubusercontent.com/1812955/98626655-2ee96b00-22e0-11eb-9d4a-c3918b21b55a.jpg" width="400" />|
|New Layout - **Large**|<img src="https://user-images.githubusercontent.com/1812955/98626672-36107900-22e0-11eb-88d1-0ae066dfba98.jpg" width="400" />|
|Members list and See All|<img src="https://user-images.githubusercontent.com/1812955/98626618-1bd69b00-22e0-11eb-81f2-3e3edba92b15.gif" width="400" />|




# Tests
